packages/pds/src/api/com/atproto/server/createInviteCode.ts

```diff
@@ -1,30 +1,22 @@
-import * as crypto from '@atproto/crypto'
-import * as uint8arrays from 'uint8arrays'
 import { Server } from '../../../../lexicon'
 import AppContext from '../../../../context'
+import { genInvCode } from './util'
 
 export default function (server: Server, ctx: AppContext) {
   server.com.atproto.server.createInviteCode({
     auth: ctx.adminVerifier,
     handler: async ({ input, req }) => {
-      const { useCount } = input.body
+      const { useCount, forAccount = 'admin' } = input.body
```

Comment:
We might want to have an idea about how this will look if/when any normal account might be marked as an administrator, when we upgrade admin auth.  I guess it's possible `forAccount` could just be attributed to the admin themselves by their did!